### DISCUSSION TITLE
Proposal and amendment

### DISCUSSION THREAD

**Discussion by salahadoroobi** [tnodir/fort#219] [Ideas]
First
I hope the Arabic language is narrowed.
Second
Arjaur Increased Construction Of Kindergartens Right Of Programs On Full Proximity Control And I Will Share The problem
Third
I am ready to be translated into Arabic.

![image](https://github.com/tnodir/fort/assets/119267418/fbbc565b-c86f-4840-a67e-d5b662d0b74c)

That's just the number you can only generate.
![image](https://github.com/tnodir/fort/assets/119267418/94bb0163-22a5-4a2e-ae03-51c561a87ec7)

**Comment by tnodir** [comment]
@salahadoroobi Please translate in the [Crowdin](https://crowdin.com/project/fort-firewall/ar).

**Reply by tnodir** [reply]
> That's just the number you can only generate.

I don’t understand the proposal. Can you describe it in Arabic maybe?

**Reply by tnodir** [reply]
> That's just the number you can only generate.

If you mean, that the maximum number of App Groups is 16, then it’s correct. I’m going to change it to 32 later.

**Reply by salahadoroobi** [reply]
نعم ولماذا لا  انه جميل 32  ولاكن لو تدعه يكون 64 الس اجمل بذالك

**Reply by salahadoroobi** [reply]
قلت انا لوما لا يكون بالغة العربية سوف يكون اجمل انا متاكد ياسيدي

**Reply by tnodir** [reply]
Should we show 64 groups in the tray context menu too? It will not be beautiful, will it?

**Reply by tnodir** [reply]
@salahadoroobi You can use [Crowdin](https://crowdin.com/project/fort-firewall/ar) to translate Fort Firewall’s texts to Arabic.

Do you have questions about Crowdin maybe?

You have to register in Crowdin.
